nixos/doc/manual/release-notes/rl-2311.section.md

```diff
@@ -125,3 +125,5 @@ The module update takes care of the new config syntax and the data itself (user
   ./common/auto-format-root-device.nix ];` When you use the systemd initrd, you
   can automatically format the root device by setting
   `virtualisation.fileSystems."/".autoFormat = true;`.
+
+- `emscriptenPackages` and `buildEmscriptenPackage` have been replaced with `pkgsCross.emscripten` and `pkgsCross.emscripten.stdenv.mkDerivation`.
```

Comment:
don't add directly at the end but at a random location to prevent merge conflicts